### PR TITLE
Gh55

### LINKED ISSUE #55: Process highlight # for activity stream session pings
Refers to [#1877](https://github.com/mozilla/activity-stream/issues/1877)

### PR DESCRIPTION
**Pull request by ncloudioj**
This closes #55 and #56

### CODE REVIEW

File: infernyx/rules.py [hunk 1]
```diff
@@ -384,6 +384,29 @@ def clean_activity_stream_performance(parts, params):
...
+        assert parts["event"]
+        assert parts['source']
+        # the client might send a unix timestamp sometimes, flag it as invalid by using a negetive number
+        if parts["value"] > 2 ** 32:
+            parts["value"] = -1
+
+        # check those optional fields
+        for f in ['experiment_id', 'session_id']:
+            # populate the optional fields with default values if they are missing
+            if f not in parts:
```
Comment (rlr) [inline infernyx/rules.py:403]: I wish JS had this 😉

**Review by rlr — APPROVED** [review]
it all makes sense. I didn't run tests but I trust you did 😆

File: infernyx/rules.py (hunk 1 above)
Comment (ncloudioj) [reply infernyx/rules.py:403]: Heh, make a wish. Dreams will come true in ES2018 😼

**Pull request merged**